### PR TITLE
[To Main] Upgrading the version of met-formio

### PR DESCRIPTION
**Pull request by VineetBala-AOT**
Issue #: https://apps.itsm.gov.bc.ca/jira/browse/DESENG-596

*Description of changes:*
- Upgrading the version of met-formio to add a default description for the form components.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: CHANGELOG.MD [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## May 6, 2024
+
+- **Feature** Auto-populate question descriptions [🎟️ DESENG-596](https://apps.itsm.gov.bc.ca/jira/browse/DESENG-596)
```
Comment (Baelx) [inline CHANGELOG.MD:3]: From the Changelog, and by merging into Main, it looks like this feature is complete just by upgrading the version. Is that the case?

**Review by Baelx — APPROVED** [review]
To me, it's not super clear if this feature is done or not. It may create confusion for folks looking in the history. 

If you're wanting to do some preparatory work for this ticket by first upgrading the library in the main branch, I'd change the Changelog entry to read something like:

`- Prepare for DESENG-596 by upgrading the met-formio library`

This would make it clear that the feature still needs to be completed but that you're doing work to prepare for it.

You're welcome to merge in any case.

File: CHANGELOG.MD (hunk 1 above)
Comment (VineetBala-AOT) [reply CHANGELOG.MD:3]: Yes the changes were pushed to met-formio and published to npm. This version of met-formio has the changes needed for this ticket. I update the changelog to have more details.

Comment (Baelx) [reply CHANGELOG.MD:3]: Ohhh okay so the substantive changes were in the met-formio repository for this. Thanks for clarifying

**Pull request merged**